Hibernate/hibernateAnnotationDemo/StudentDemo.java

```diff
@@ -0,0 +1,30 @@
+package hibernateAnnotationDemo;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class StudentDemo {
+
+	public static void main(String[] args) {
+		SessionFactory factory = new Configuration()
+				.configure("hibernate.cfg.xml")
+				.addAnnotatedClass(Student.class)
+				.buildSessionFactory();
+
+		Session session = factory.getCurrentSession();
+		try {			
+			System.out.println("Creating new student object...");
+			Student tempStudent = new Student("Ram", "kumar", "ram@gmail.com");
```

Comment:
Try with the following input your student name with something that is greater than 256 characters and see how to fix the exception encountered